### DISCUSSION TITLE
Make the most common updates available in the sidebar area

### DISCUSSION THREAD

**Discussion by ILW8** [ppy/osu#19103] [Ideas]
5. In response to a request for feedback from running the lazer tournament client for [The Roundtable](https://osu.ppy.sh/wiki/en/Tournaments/The_Roundtable)
_____
It would be nice to be able to update bracket scores/switch to bracket editor without losing focus of the gameplay/mappool. For example, update different bracket scores without switching to the bracket screen: useful for when multiple matches are being played at the same time. The person running the tourney client can update other brackets' scores without switching away from gameplay screen. That way, when the broadcast does switch to the bracket screen, scores will be already updated.

One other thing has to  do with updating the bracket nodes themselves. Perhaps specific to lan environments, we have to have players on  a certain slot due to IRL camera placement. Because of that, more than once did we have to swap the players from left to right of the tournament client. Would also be nice to  be able to do that during gameplay instead of having to switch to casters while make changes to brackets, but I understand this  is more  of an edge case  that stems from miscommunication between the IRL staff and the remote staff.

**Comment by peppy** [comment]
Initial thoughts are that the easiest way to implement this would be to allow two clients to run at the same time, and add file watching on `bracket.json`. Currently the file is only read on startup, but due to the way it is implemented (everything-is-bindable) it should be feasible to read a new version of the json as an in-place copy into the existing object and have things just-work(tm).

**Reply by Soetch** [reply]
But wouldn't it be eating more performances ? (Writing it again)
Because the "double-client" idea seems feasible, but they'll have to communicate and things.

So one would be the "updating" and the other, the "updated" ?
